charts/kubeclarity/templates/deployment.yaml

```diff
@@ -38,7 +38,7 @@ spec:
           resources:
 {{- toYaml .Values.kubeclarity.initContainers.resources | nindent 12 }}
         - name: '{{ include "kubeclarity.name" . }}-wait-for-sbom-db'
-          image: curlimages/curl:7.84.0
+          image: {{ index .Values "curl" "image" "registry" | default "docker.io" }}/{{ index .Values "curl" "image" "repository" | default "curlimages/curl" }}:{{ index .Values "curl" "image" "tag" | default "7.84.0" }}
```

Comment:
I think we can safely rely on the defaults in the charts values.yaml, I don't think we need to complicate the template here with all the "default ..."'s and then have to maintain the version in two different places.